frontend/scripts/robot/actions/edit.js

```diff
@@ -9,7 +9,7 @@ let Robot = require("frontend/robot/models");
 export default function edit(model) {
   let newModel = Robot(model);
   let id = newModel.id;
-  let oldModel = State.select("robots", "models", id);
+  let oldModel = State.select("robots", "models", id).get();
```

Comment:
This was a bug: we got cursor here instead of object and rollback in case of error didn't work